buildSrc/src/main/groovy/org/elasticsearch/gradle/BuildPlugin.groovy

```diff
@@ -215,6 +215,7 @@ class BuildPlugin implements Plugin<Project> {
             project.rootProject.ext.inFipsJvm = inFipsJvm
             project.rootProject.ext.gradleJavaVersion = JavaVersion.toVersion(gradleJavaVersion)
             project.rootProject.ext.java9Home = "${-> findJavaHome("9")}"
+            project.rootProject.ext.defaultParallel = findDefaultParallel(project.rootProject)
```

Comment:
Why set it up as an extension property rather than just return it ? 
I think we should wait for a use-case that needs it.

Reply:
I wanted to build it one time. I could keep it as a private variable to the plugin if you think that is more idiomatic.